### PR TITLE
Patch helper-zcl to return the cluster count for bitmaps and enums

### PR DESCRIPTION
**Pull request by hasty**
This updates the ZCL helper functions to return whether a given enum or bitmap is in use by more than one cluster  (excluding Features bitmaps). This is to support sharing bitmaps and enums between clusters, much as structs can be currently shared.

### CODE REVIEW

**Review by paulr34 — APPROVED** [review]
Looks fine to me, and matter codegen passed. 
Approving to allow you to merge when ready.

**Comment by hasty** [comment]
I don't have write access to the repo; could someone who does merge this?

**Comment by paulr34** [comment]
> I don't have write access to the repo; could someone who does merge this?

done

**Pull request merged**
